src/profile-logic/import/linux-perf.js

```diff
@@ -41,8 +41,11 @@ export function convertPerfScriptProfile(
     const markers = {
```

Comment:
I meant to make this comment during my review but forgot: to use the GeckoProfile type instead of MixedObject, so that we're sure we abide by the version 24 of the format. So I did that myself and fixed the differences. That should be pretty much straightforward.